source/meta.tex

```diff
@@ -2029,6 +2067,28 @@
  are complete types, \cv{}~\keyword{void}, or arrays of
  unknown bound.\\ \rowsep
 
+\tcode{template<class Fn, class Tuple>}\br
+ \tcode{struct \libglobal{apply_result};}
+ &
+ If the expression \tcode{\placeholdernc{INVOKE}(declval<Fn>(), \placeholdernc{ELEMS-OF}(Tuple)...)}\iref{func.require}
+ is well-formed when treated as an unevaluated operand\iref{term.unevaluated.operand},
+ the member typedef \tcode{type} denotes the type
+ \tcode{decltype(\placeholdernc{INVOKE}(declval<Fn>(), \placeholdernc{ELEMS-OF}(Tuple)...))};
+ otherwise, there shall be no member \tcode{type}. Access checking is
+ performed as if in a context unrelated to \tcode{Fn} and
+ \tcode{Tuple}. Only the validity of the immediate context of the
+ expression is considered.
+ \begin{note}
+ The compilation of the expression can result in side effects such as
+ the instantiation of class template specializations and function
+ template specializations, the generation of implicitly-defined
+ functions, and so on. Such side effects are not in the ``immediate
+ context'' and can result in the program being ill-formed.
+ \end{note}
+ \expects \tcode{Fn} and \tcode{Tuple}
+ are complete types, \cv{}~\keyword{void}, or arrays of
+ unknown bound.\\ \rowsep
```

Comment:
It would be nice to have semantic line breaks in this area, at least for new sentences.

Reply:
Agreed. It was copy+pasted from `is_invocable` but I'll reflow it.